### PR TITLE
updates to AppFactory

### PR DESCRIPTION
**Pull request by snopoke**
@kaapstorm 

cc @TylerSheffels

### CODE REVIEW

**Comment by snopoke** [comment]
This is all test changes, no actual code changes

File: corehq/apps/app_manager/tests/test_form_workflow.py [hunk 1]
```diff
@@ -1,12 +1,10 @@
 import copy
 from django.test import SimpleTestCase
 from mock import patch
-from corehq.apps.app_manager.const import APP_V2
-from corehq.apps.app_manager.models import WORKFLOW_FORM, FormLink, AdvancedOpenCaseAction, OpenCaseAction, \
-    AdvancedModule, Application, Module, UpdateCaseAction, LoadUpdateAction, OpenSubCaseAction, FormActionCondition, \
-    WORKFLOW_MODULE, WORKFLOW_ROOT
+from corehq.apps.app_manager.const import AUTO_SELECT_RAW
+from corehq.apps.app_manager.models import WORKFLOW_FORM, FormLink, WORKFLOW_MODULE, WORKFLOW_ROOT, WORKFLOW_PREVIOUS
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_form_workflow.py]: E501 line too long (117 > 115 characters)

File: corehq/apps/app_manager/tests/test_form_workflow.py [hunk 2]
```diff
@@ -42,138 +22,54 @@ def tearDown(self):
...
+        m1, m1f0 = factory.new_basic_module('m1', 'frog')
+        factory.form_updates_case(m1f0)
 
         m0f0.post_form_workflow = WORKFLOW_FORM
         m0f0.form_links = [
             FormLink(xpath='true()', form_id=m1f0.unique_id)
         ]
 
-        self.assertXmlPartialEqual(self.get_xml('form_link_create_update_case'), app.create_suite(), "./entry[1]")
+        self.assertXmlPartialEqual(self.get_xml('form_link_create_update_case'), factory.app.create_suite(), "./entry[1]")
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_form_workflow.py]: E501 line too long (122 > 115 characters)

File: corehq/apps/app_manager/tests/test_form_workflow.py [hunk 3]
```diff
@@ -235,87 +102,76 @@ def test_link_to_child_module(self):
...
-        m2f1.post_form_workflow = WORKFLOW_FORM
-        m2f1.form_links = [
-            FormLink(xpath="true()", form_id=m1f1.unique_id),
+        m2f0.post_form_workflow = WORKFLOW_FORM
+        m2f0.form_links = [
+            FormLink(xpath="true()", form_id=m1f0.unique_id),
         ]
 
-        self.assertXmlPartialEqual(self.get_xml('form_link_child_modules'), app.create_suite(), "./entry[3]")
+        self.assertXmlPartialEqual(self.get_xml('form_link_child_modules'), factory.app.create_suite(), "./entry[3]")
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_form_workflow.py]: E501 line too long (117 > 115 characters)

**Comment by snopoke** [comment]
@TylerSheffels bump

**Comment by kaapstorm** [comment]
:+1: Nice!

**Comment by TylerSheffels** [comment]
Whoops, forgot to review before.  Looks good.

**Pull request merged**